.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node.js
+node_js:
```

Comment:
Cool that you setup eslint and travis, did you manage to get it working / was it useful?